List-of-Free-Code-Camp-city-based-Campsites.md

```diff
@@ -465,6 +465,7 @@ If your city isn't listed here, you should [[create a Campsite for your city|How
   - Tamil Nadu
     - [Chennai](https://www.facebook.com/groups/free.code.camp.chennai/)
     - [Coimbatore](https://www.facebook.com/groups/free.code.camp.coimbatore)
+    - [Madurai](https://www.facebook.com/groups/free.code.camp.maduraicity/)
```

Comment:
@hallaathrad ah! I didn't know one can't change the url once created. I thought I would point out and then you can follow it up and ask the admin to modify the url.
It's alright then. I won't comment about the url. :+1: